src/com/houarizegai/calculator/Calculator.java

```diff
@@ -17,10 +13,31 @@
     private static final int MARGIN_X = 20;
     private static final int MARGIN_Y = 60;
 
-    private JFrame window; // Main window
-    private JTextField inText; // Input text
-    private JButton btnC, btnBack, btnMod, btnDiv, btn7, btn8, btn9,
-            btnMul, btn4, btn5, btn6, btnSub, btn1, btn2, btn3, btnAdd, btnPoint, btn0, btnEqual, btnRoot, btnPower, btnLog, btnSwitchThemes, btnSwitchToScientificMode;
```

Comment:
really? the above declarations can be simple split into several lines just to stick to the rule of
no more than 80 characters per line